packages/react-magma-dom/src/components/TreeView/TreeView.test.js

```diff
@@ -330,6 +334,29 @@ describe('TreeView', () => {
       userEvent.click(getByTestId('item2-itemwrapper'));
       expect(onExpandedChange).not.toHaveBeenCalled();
     });
+
+    it('should work correctly with expandedAll() function', () => {
+      const onExpandedChange = jest.fn();
+      const apiRef = React.createRef();
+      const { getByText } = render(
+        <>
+          <button onClick={() => apiRef.current.expandAll()}>Expand All</button>
+          {getTreeItemsWithDisabled({
+            selectable: TreeViewSelectable.single,
+            onExpandedChange,
+            apiRef,
+          })}
+        </>
+      );
+
+      expect(getByText('Expand All')).toBeInTheDocument();
+
+      userEvent.click(getByText('Expand All'));
+
+      expect(onExpandedChange).toHaveBeenCalled();
+      expect(onExpandedChange).toHaveBeenCalledTimes(1);
```

Comment:
This is repetitive, we don't need `toHaveBeenCalled` if we use `toHaveBeenCalledTimes(1)`